src/Language/PureScript/Docs/RenderedCode/Render.hs

```diff
@@ -0,0 +1,172 @@
+
+-- | Functions for producing RenderedCode values from PureScript Type values.
+
+module Language.PureScript.Docs.RenderedCode.Render (
```

Comment:
Maybe later we can try to unify this with the existing `prettyPrintType` function. We could maybe write a "plain text renderer".